examples/llama2/tensor_parallel_generate.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
What is this used for?

Reply:
Pulled out the change to OptimumMerge & OptimumModelHandler. PR focus on introducing merge decoders for OnnxConversion only.